common/system.h

```diff
@@ -451,6 +451,12 @@ class OSystem : Common::NonCopyable {
 		 */
 		kFeatureCursorPalette,
 
+		/**
+		 * Backends supporting this feature allow cursors to contain an alpha
```

Comment:
I do not quite understand why this global flag is needed. Could you please elaborate?

And if it is needed, then a relevant test must be added to the testbed engine.

Reply:
SDL doesn't allow alpha blits (or true colour sources IIRC) to a paletted surface, so it's necessary to get the GUI to use the CLUT8 cursor instead. The RS90 backend avoided this issue because USE_RGB_COLOR was disabled, but on RISC OS both true colour and palette only modes area available in a single build, so a feature flag is required instead.

I'll see what I can do about adding alpha cursors to the testbed engine.

Reply:
I've added a new test case for this.